presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/RaptorMetadata.java

```diff
@@ -175,6 +175,11 @@ private RaptorTableHandle getTableHandle(SchemaTableName tableName)
         }
         List<TableColumn> tableColumns = dao.listTableColumns(table.getTableId());
         checkArgument(!tableColumns.isEmpty(), "Table %s does not have any columns", tableName);
+        Optional<List<String>> bucketAssignments = Optional.empty();
+
+        if (table.getDistributionId().isPresent()) {
```

Comment:
If we change `distributionId` to `Optional<Long>`, then we use `Optional.map()` here